helm-chart/utils/postgres_migrations/version_upgrades/README.md

```diff
@@ -0,0 +1,48 @@
+# Upgrading postgresql
+
+Here we describe how the version of a postgresql instance which is installed through the Renku chart can be upgraded. While minor version upgrades are unproblematic and do not need any manual intervention, major version upgrades require some care. There are two recommended ways of performing this task:
+
+1) Dump all the data in the old instance using `pd_dumpall`, start a fresh instance using the target version and import the previous data dump.
+
+2) Using the [pg_upgrade](https://www.postgresql.org/docs/devel/pgupgrade.html) command.
+
+While `pg_upgrade` would achieve this task with much shorter downtime and if desired without copying any data, it seems a bit more complicated to use in a k8s context as one needs to run both versions (old and new postgresql) alongside in one container. We therefore describe option number 1) here which seems simple enough for databases of moderate size.
+
+## Preparation
+
+- Modify the example manifest files in this folder according to your needs. You can use the followig command to figure out the total amount of data currently stored in your database. This will give you a fair estimate of the size of your data dump.
+
+```sql
+SELECT pg_database.datname as "database_name", pg_database_size(pg_database.datname)/1024/1024 AS size_in_mb FROM pg_database ORDER by size_in_mb DESC;
```

Comment:
We can replace this with a clear way of how to access the postgres instance. Or at least add it as a suggestion.
```bash
```bash
kubectl -n renku exec -ti renku-postgresql-0 -- bash
PGUSER=$POSTGRES_USER PGPASSWORD=$POSTGRES_PASSWORD psql -c 'SELECT pg_database.datname as "database_name", pg_database_size(pg_database.datname)/1024/1024 AS size_in_mb FROM pg_database ORDER by size_in_mb DESC;'
```
```

Reply:
done